boards/nxp/fmuk66-v3/default-boardconfig

```diff
@@ -0,0 +1,241 @@
+
+#
+# Platform: nuttx
+#
+
+#
+# Vendor: nxp
+#
+
+#
+# Model: fmuk66-v3
+#
+
+#
+# Label: default
+#
+
+#
+# Toolchain: Toolchain-arm-none-eabi
+#
+
+#
+# Architecture: cortex-m4
+#
+
+#
+# Romfsroot: px4fmu_common
+#
+
+#
+# drivers
+#
+
+#
+# ADC
+#
+# CONFIG_DRIVERS_ADC_ADS1115 is not set
+CONFIG_DRIVERS_ADC_BOARD_ADC=y
+# end of ADC
+
+CONFIG_DRIVERS_BAROMETER=y
+# CONFIG_DRIVERS_BAROMETER_BMP280 is not set
+# CONFIG_DRIVERS_BAROMETER_BMP388 is not set
+# CONFIG_DRIVERS_BAROMETER_DPS310 is not set
+# CONFIG_DRIVERS_BAROMETER_LPS22HB is not set
+# CONFIG_DRIVERS_BAROMETER_LPS25H is not set
+# CONFIG_DRIVERS_BAROMETER_LPS33HW is not set
+CONFIG_DRIVERS_BAROMETER_MPL3115A2=y
+# CONFIG_DRIVERS_BAROMETER_MS5611 is not set
+# CONFIG_DRIVERS_BAROMETER_TCBP001TA is not set
+CONFIG_DRIVERS_BATT_SMBUS=y
+# CONFIG_DRIVERS_BOOTLOADERS is not set
+CONFIG_DRIVERS_CAMERA_CAPTURE=y
+CONFIG_DRIVERS_CAMERA_TRIGGER=y
+CONFIG_DRIVERS_DIFFERENTIAL_PRESSURE=y
+CONFIG_DRIVERS_DISTANCE_SENSOR=y
+# CONFIG_DRIVERS_DSHOT is not set
+CONFIG_DRIVERS_GPS=y
+# CONFIG_DRIVERS_HEATER is not set
+
+#
+# IMU
+#
+# CONFIG_DRIVERS_IMU_ADIS16477 is not set
+# CONFIG_DRIVERS_IMU_ADIS16497 is not set
+# CONFIG_DRIVERS_IMU_ANALOG_DEVICES is not set
+# CONFIG_DRIVERS_IMU_BOSCH is not set
+CONFIG_DRIVERS_IMU_FXAS21002C=y
```

Comment:
We should see what this is like compressed (what NuttX does).

Reply:
Solved it by actually calling defconfig which creates a compressed file
https://github.com/PX4/PX4-Autopilot/blob/6d76af9fb08b2c3d4dccccdc357ed143d5b885c9/cmake/kconfig.cmake#L30-L33